CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 9.10.1
```

Comment:
@kevmoo I bumped this off dev as need to update flutter/cocoon with this fix. Is this ready to be released, or should I keep it marked as dev?

Reply:
I have no problem if you want to publish now.